Composer/packages/server/src/services/storage.ts

```diff
@@ -18,6 +18,9 @@ class StorageService {
 
   constructor() {
     this.storageConnections = Store.get(this.STORE_KEY);
+    this.storageConnections.forEach(s => {
+      this.createFolderRecurively(s.defaultPath);
+    });
```

Comment:
What if the user has changed their default folder and then deleted the one we create? Won't this create /Documents/Composer every time?

Reply:
the default path is set in file data.template.json. It does not allow users to change it. These lines of code it to make sure the default path is always valid.